pkg/apiserver/registry/networkpolicy/clustergroupmember/rest.go

```diff
@@ -44,15 +44,26 @@ type groupMembershipQuerier interface {
 	GetGroupMembers(name string) (controlplane.GroupMemberSet, []controlplane.IPBlock, error)
 }
 
+type paginationInfo struct {
+	pageNumber int64 // stores current page number from query
+	pageLimit  int64 // stores page limit from query
+}
+
 func (r *REST) New() runtime.Object {
 	return &controlplane.ClusterGroupMembers{}
 }
 
-func (r *REST) Get(ctx context.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
+func (r *REST) Get(ctx context.Context, name string, options runtime.Object) (runtime.Object, error) {
```

Comment:
Yes it would not be `nil`, also found that it is initiated [here](https://github.com/kubernetes/apiserver/blob/master/pkg/endpoints/handlers/get.go#L115) in K8s api server get.

Reply:
Sure, cleaned it up.